internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -292,6 +302,16 @@ func verifyGoogleCloudLogging(gcl *loggingv1.GoogleCloudLogging) bool {
 	return i == 1
 }
 
+//func verifyAzureMonitorLog(name string, azml *loggingv1.AzureMonitor) (bool, status.Condition) {
```

Comment:
??

Reply:
Disabled validation until we agreed spec